Tests/ProjectDescriptionTests/TuistConfigTests.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import XCTest
+@testable import ProjectDescription
+
+final class TuistConfigTests: XCTestCase {
+    func test_tuistconfig_toJSON() throws {
+        let tuistConfig = TuistConfig(generationOptions:
+            [.generateManifest,
+             .xcodeProjectName("someprefix-\(.projectName)")])
+
+        let encoded = try JSONEncoder().encode(tuistConfig)
```

Comment:
What about simplifying the test by using [this](https://github.com/tuist/tuist/blob/master/Sources/TuistCoreTesting/Extensions/XCTestCase+Extras.swift#L76)